layouts/_default/_markup/render-heading.html

```diff
@@ -0,0 +1 @@
+{{ template "_default/_markup/td-render-heading.html" . }}
```

Comment:
could you give this a comment explaining what it does? AIUI this is a hugo rendering hook?

Links to https://gohugo.io/render-hooks/headings/ and https://www.docsy.dev/docs/adding-content/navigation/#heading-self-links might be useful.

Reply:
Comment added.